fabric-biomes-v1/src/main/java/net/fabricmc/fabric/impl/biome/InternalBiomeUtils.java

```diff
@@ -138,7 +148,7 @@ public static void injectBiomesIntoClimate(LayerRandomnessSource random, int[] v
 		if (reqWeightSum < vanillaArray.length) {
 			// Vanilla biome; look it up from the vanilla array and transform accordingly.
 
-			result.accept(transformBiome(random, BuiltinRegistries.BIOME.get(vanillaArray[(int) reqWeightSum]), climate));
+			result.accept(transformBiome(random, Biomes.fromRawId(vanillaArray[(int) reqWeightSum]), climate));
```

Comment:
I would suggest using `InternalBiomeUtils.fromRawId` similar to `InternalBiomeUtils.getRawId` so we can make them both use a DynamicRegistryManager if needed

Reply:
Vanilla still uses Biomes.fromRawId itself in a place where a DynamicRegistryManager is unavailable, so this won't help.

Reply:
You mean `BiomeLayer.sample`? I would've changed that too.
Dunno, to me that would feel better.